socs/agents/ucsc_radiometer/agent.py

```diff
@@ -0,0 +1,122 @@
+from os import environ
+
+import requests
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import Pacemaker, TimeoutLock
+
+
+class UCSCRadiometerAgent:
+    """Monitor the PWV Flask Server.
+
+    Parameters
+    ----------
+    agent : OCS Agent
+        OCSAgent object which forms this Agent
+    url : str
+        url of the flask server on the internet
+    year : int
+        year for the corresponding Julian Day
+    """
+
+    def __init__(self, agent, url, year):
+        self.agent = agent
+        self.log = agent.log
+        self.lock = TimeoutLock()
+
+        self.url = url
+        self.year = year
+
+        self.take_data = False
+
+        agg_params = {'frame_length': 60,
+                      'exclude_influx': False}
+
+        # register the feed
+        self.agent.register_feed('pwvs',
+                                 record=True,
+                                 agg_params=agg_params,
+                                 buffer_time=1
+                                 )
+
+        self.last_published_reading = None
+
+    @ocs_agent.param('test_mode', default=False, type=bool)
+    def acq(self, session, params=None):
+        """acq()
+
+        **Process** - Fetch values from PWV Flask Server
+
+        Parameters
+        ----------
+        test_mode : bool, option
+            Run the Process loop only once. Meant only for testing.
+            Default is False.
+        """
+        pm = Pacemaker(1 / 60, quantize=False)
+
+        self.take_data = True
+        while self.take_data:
+            pm.sleep()
+
+            r = requests.get(self.url)
+            data = r.json()
+            last_pwv = data['pwv']
+            last_timestamp = data['timestamp']
+
+            pwvs = {'block_name': 'pwvs',
+                    'timestamp': last_timestamp,
+                    'data': {'pwv': last_pwv}
+                    }
+
+            if self.last_published_reading is not None:
+                if last_timestamp > self.last_published_reading[1]:
+                    self.agent.publish_to_feed('pwvs', pwvs)
+                    self.last_published_reading = (last_pwv, last_timestamp)
+            else:
+                self.agent.publish_to_feed('pwvs', pwvs)
+                self.last_published_reading = (last_pwv, last_timestamp)
+
+            if params['test_mode']:
+                break
+
+        return True, 'Acquisition exited cleanly.'
+
+    def _stop_acq(self, session, params=None):
+        """
+        Stops acq process.
+        """
+        self.take_data = False
+        return True, 'Stopping acq process'
+
+
+def add_agent_args(parser_in=None):
+    if parser_in is None:
+        from argparse import ArgumentParser as A
+        parser_in = A()
```

Comment:
Move the argparse import to the top of the file as just `import argparse` and make these lines:
```
    if parser is None:
        parser = argparse.ArgumentParser()
```
for consistency with other agents. (Also make the argument `parser` instead of `parser_in`.)